docs/notebooks/ordinal_regression.ipynb

```diff
@@ -0,0 +1,1695 @@
+{
```

Comment:
<p>here you could also use the ppc function that I shared</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>